### DISCUSSION TITLE
Andor sCMOS camera added to bonsai workflow?

### DISCUSSION THREAD

**Discussion by emp2163** [bonsai-rx/bonsai#1559] [Q&A]
Hi,
I am interested in using bonsai to orchestrate flir cameras, a rotary encoder, etc. and an Andor Zyla camera. Has anyone else added an Andor camera to their bonsai workflow? It can send out TTL signals. Can anyone point me in the direction of a guide for adding this hardware? Or, if someone has tried it and andor zyla cameras are not compatible please let me know!
Best wishes, Emily

**Comment by bruno-f-cruz** [comment]
This is my personal opinion. After a quick look I found their SDK. This SDK is not only not open-source, but it also seems that you actually have to pay for it. In my view, this is a huge turn-off when it comes to making a decision on whether a hardware interface is worth implementing in an open-source ecosystem like bonsai, or not. 

[See SDK here](https://andor.oxinst.com/products/software-development-kit/software-development-kit)

**Reply by emp2163** [reply]
Thank you so much for your swift reply! I do have the SDK in hand. I used the sdk to run the camera in matlab but had frame drops on that platform so interested in using something c++ based instead. But sounds like you are less enthusiastic about using bonsai given my camera and system. Andor support is also not great but the zyla camera is the camera I have access to use.
